install.sh

```diff
@@ -12,26 +12,44 @@ opkg update
 opkg install curl python-light python-pip pyOnionI2C
 pip install flask
 
+echo "Linking new libraries"
 # add the hackerschool to the site packages (only for old code)
-if [ -L /usr/lib/python2.7/site-packages/hackerschool ]; then
+if [[ -L /usr/lib/python2.7/site-packages/hackerschool ]]; then
     rm /usr/lib/python2.7/site-packages/hackerschool
 fi
 ln -s "/opt/hucon/python_lib" /usr/lib/python2.7/site-packages/hackerschool
 
 # add the hucon to the site packages
-if [ -L /usr/lib/python2.7/site-packages/hucon ]; then
+if [[ -L /usr/lib/python2.7/site-packages/hucon ]]; then
     rm /usr/lib/python2.7/site-packages/hucon
 fi
 ln -s "/opt/hucon/python_lib" /usr/lib/python2.7/site-packages/hucon
 
-# add the init script as symbolic link
-chmod +x init.d/i2c_led
-chmod +x i2c_led.sh
-if [ ! -f /etc/init.d/i2c_led ]; then
-    ln -s "/opt/hucon/init_d/i2c_led" /etc/init.d/i2c_led
+echo "Removing old server startup"
+sed -i '/start_server.sh/d' /etc/rc.local
+
+if [[ -e /etc/init.d/HuconWebserver ]]; then
+    /etc/init.d/HuconWebserver disable
+    rm /etc/init.d/HuconWebserver
 fi
-/etc/init.d/i2c_led enable
 
-# add the server to start at boot
-sed -i '/start_server.sh/d' /etc/rc.local
-sed -i "/^exit 0/i sh /opt/hucon/start_server.sh >> /tmp/hucon.log 2>&1 & " /etc/rc.local
+echo "Deploying new server startup"
+cp /opt/hucon/init.d/HuconWebserver /etc/init.d/HuconWebserver
+chmod +x /etc/init.d/HuconWebserver
+/etc/init.d/HuconWebserver enable
+
+echo "Removing old i2c startup"
+if [[ -e /etc/init.d/I2C_led ]] || [[ -e /etc/init.d/I2C_led ]]; then
```

Comment:
Did you mean the following:
`if [[ -e /etc/init.d/i2c_led ]] || [[ -e /etc/init.d/I2C_led ]]; then`

**Both checks are uppercase**

Reply:
yes you are right - pasta in the copier is not good xD